rotkehlchen/exchanges/kraken.py

```diff
@@ -1146,7 +1061,23 @@ def history_event_from_kraken(
 
                 # Make sure to not generate an event for KFEES that is not of type FEE
                 if asset != A_KFEE:
-                    event = HistoryEvent(
+                    # Process asset movements
+                    if event_type in {HistoryEventType.DEPOSIT, HistoryEventType.WITHDRAWAL}:
```

Comment:
Add to the comment that there aren't any deposit/withdrawal events of KFEEs. I've checked the tests that we have and seems to be the case. It it wasn't like this this logic would need adjusment so it is one of the assumptions